lab-matt/html/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html ng-app='cowsayApp'>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lab 21-22</title>
+    <link rel='stylesheet' type = 'text/css' href = 'bundle.css'/>
+    <script src='bundle.js'></script>
+  </head>
+  <body>
+    <main ng-controller = 'CowsayController'>
+      <form>
+        <label for='cowsay-text' type = 'text' ng-model='cowsayText'/>What Say You</form>
+        <input name='cowsay-text' type="text" ng-model='cowsayText'/>
+        <select ng-model='cowsayFile'>
+          <option ng-repeat='cow in cowfiles' value ='{{cow}}'>{{cow}}</option>
+        </select>
+        <button ng-click='final()' type='submit'>BOOYAH~</button>
```

Comment:
I would delete the type="submit" from the "previous button", since this button is not actually submitting the form.